src/command.h

```diff
@@ -454,12 +476,24 @@ class BufferCommand : public PipelineCommand {
   Buffer* buffer_ = nullptr;
   BufferType buffer_type_;
   bool is_subdata_ = false;
-  uint32_t descriptor_set_ = 0;
-  uint32_t binding_num_ = 0;
   uint32_t offset_ = 0;
   std::vector<Value> values_;
 };
 
+class SamplerCommand : public BindableResourceCommand {
```

Comment:
Add a comment above the `class` similar to the other ones.

Reply:
Done